### DISCUSSION TITLE
Possible integration/sync with Podgrab

### DISCUSSION THREAD

**Discussion by akhilrex** [advplyr/audiobookshelf#725] [Ideas]
I just saw this app on HN and this seems really interesting.

I am the maintainer of [Podgrab](https://github.com/akhilrex/podgrab)  and even though Podgrab has a rudimentary player built into it it's primarily a downloading and archiving tool. I get a lot of feature requests pertaining to streaming/playing and I think this could be a great complementary app. 

Let me set this up and see what can be done.

**Comment by advplyr** [comment]
Awesome, we are open to any and all integrations!
